src/test/RecordingMetadata/BulkEditLabels.test.tsx

```diff
@@ -107,9 +107,11 @@ const mockArchivedRecordingsResponse = {
   data: {
     targetNodes: [
       {
-        recordings: {
-          archived: {
-            data: [mockArchivedRecording] as ArchivedRecording[],
+        target: {
+          recordings: {
+            archived: {
```

Comment:
should be changed to archivedRecordings?

Reply:
I have been mostly ignoring the label editing features since these also depend on https://github.com/cryostatio/cryostat3/pull/319 . I could make some of these adjustments already here to help out with that, but the changes can't actually be tested until that other PR is ready.